README.md

```diff
@@ -16,73 +16,44 @@
+This package is aimed at begining graduate students and, hopefully, at medical staff interested in digital pathology. Please send comments and criticisms to **[tialab@dcs.warwick.ac.uk](mailto:tialab@dcs.warwick.ac.uk)**.
 
-or
+We assume some knowledge of the following topics, though, initially, it may be sufficient to glance through a brief introduction using Google.
 
-    pip install -r requirements.txt
+1. **`Anaconda`** is a distribution of the Python language containing many useful Python packages. To offset its many advantages, it's quite large (about 2Gb). But that is not too much space for a modern home computer.
+2. **`Conda`**is an app, found in Anaconda, giving convenient management of large software packages.
+3. When working on a computer, one works in a certain *environment*, in which certain commands and procedures make sense. When working on large **Python** projects, such as ours, it is often convenient to work in a *virtual environment*. These virtual environments are controlled by `conda`. 
+4. Github: the site where this repository is found.
+5. git, a language giving control of different versions of the same program. A modern novelist might find it useful for keeping different versions of a novel, but usually it's used for software development.
 
-2.Clone the tiatoolbox
-
-    git clone https://github.com/TIA-Lab/tiatoolbox.git
-
-3.Please try
-
-    python -m tiatoolbox -h
+Getting Started
+===============
 
-pip
----
+1. Make a directory (also known as a *folder*) into which you will download this repository. You can call it anything you like. We will call it **tb**.
 
-    pip install tiatoolbox
+2. Open a terminal window, while in **tb**, for inputting commands.
 
-conda
------
+3. Clone (download a copy) of  the official repository for `tiatoolbox`using 
+   `conda clone https://github.com/TIA-Lab/tiatoolbox.git` 
+   
+4. Change into the new subdirectory `tb/tiatoolbox`.
 
-    conda env create -n tiatoolbox python=3.{6,7,8}
-    conda activate tiatoolbox
-    pip install tiatoolbox
+5. Create the virtual environment `tiatoolbox` with
+   `conda env create -f requirements.conda.yml`
+   (The name `tiatoolbox` for the environment is hidden inside the requirements file.)
+   
+6. `conda activate tiatoolbox`
 
-tiatoolbox --help
-=================
+   You now have several different objects called *tiatoolbox*, among them the directory in which your terminal is operating, and the environment *tiatoolbox* , created by conda.
 
-    usage: tiatoolbox [-h] [--version] [--verbose VERBOSE]
-                     {read-bounds,
-                      save-tiles,
-                      slide_info,
-                      slide-thumbnail,
-                      stainnorm,
-                      }
-                     ...
+7. Check success with `conda list`, which lists all the Python packages in the `tiatoolbox` virtual environment and also the version of Python that will be used in that environment.
```

Comment:
I am not sure if this step is necessary.

Reply:
"conda list" no longer mentioned